core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -257,6 +260,11 @@ object AdminUtils extends Logging {
           throw new InvalidTopicException("Topic \"%s\" collides with existing topics: %s".format(topic, collidingTopics.mkString(", ")))
         }
       }
+
+      //by default we make user that issues topic creation as the owner.
+      if (!config.containsKey(LogConfig.OwnersProp)) {
+        config.put(LogConfig.OwnersProp,  System.getProperty("user.name"))
+      }
```

Comment:
I preferred this way as we can establish some sort of true identity rather than complete anonymity.  Once we have sasl support we can actually derive identities based on who the logged in sasl user is instead of asking the user to pass this as an argument or reliying on System.getProperty("user.name"). can you elaborate why it is better to just leave it as anonymous.